### PR TITLE
random-password-generator

### PR DESCRIPTION
**Pull request by navbytes**
## Description

Initial commit to generate random passwords

## Checklist

- [ x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ x] I checked that files in the `assets` folder are used by the extension itself
- [ x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/random-password-generator/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "name": "random-password-generator",
+  "title": "Random Password Generator",
+  "description": "Generate random password of different length",
+  "icon": "icon.png",
+  "author": "textnav",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Generate random password",
```
Comment (fe9lix) [inline extensions/random-password-generator/package.json:12]:
```suggestion
      "title": "Generate Random Password",
```

File: extensions/random-password-generator/src/Action.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { ActionPanel, Action } from "@raycast/api";
+
+export function Actions({ password, setShowingDetails }: { password: string; setShowingDetails: () => void }) {
+  return (
+    <ActionPanel title={password}>
+      <ActionPanel.Section>
+        <Action.CopyToClipboard content={password} title="Copy Password" shortcut={{ modifiers: ["cmd"], key: "." }} />
+        <Action title="Toggle Detail" onAction={() => setShowingDetails()} />
```
Comment (fe9lix) [inline extensions/random-password-generator/src/Action.tsx:8]: Could you assign an icon here? (Either all actions should have an icon or none of them, also see the [store guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store).)

File: extensions/random-password-generator/package.json (hunk 1 above)
Comment (mattisssa) [reply extensions/random-password-generator/package.json:12]: @fe9lix let's commit this kind of change into the PR straight away.

File: extensions/random-password-generator/src/Action.tsx (hunk 2 above)
Comment (textnav) [reply extensions/random-password-generator/src/Action.tsx]: sure

**Comment by textnav** [comment]
> hi @textnav 👋 Nice work! 💪 CI seems to be failing because the images don't conform to the [guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots). Mind updating them? 😸

sure

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @textnav 👋 

Thanks for the corrections and your extension! 🔥 

Your metadata images are not following the [guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots), can you look into this? If you need help, let us know 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by textnav** [comment]
> Hi @textnav 👋
> 
> Thanks for the corrections and your extension! 🔥
> 
> Your metadata images are not following the [guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots), can you look into this? If you need help, let us know 😄
> 
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

🆗 ✅

**Comment by sxn** [comment]
🎉

**Pull request merged**